requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy==1.24.4
 monty==2023.5.8
-pymatgen==2023.06.23
+pymatgen==2023.07.20
```

Comment:
can you remove this pin, we will have to wait to the next release so the symmetry errors are fixed